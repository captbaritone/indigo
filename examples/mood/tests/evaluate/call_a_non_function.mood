pub fn test(oops: f64): f64 {
    oops()
}