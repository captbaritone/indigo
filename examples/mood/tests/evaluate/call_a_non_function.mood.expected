-----------------
INPUT
----------------- 
pub fn test(oops: f64): f64 {
    oops()
}
-----------------
OUTPUT
-----------------
Error: Tried calling "oops", but "oops" is not a function.:
 --> call_a_non_function.mood:2:5

  | pub fn test(oops: f64): f64 {
2 |     oops()
  |     ^^^^^^ "oops" is not a function.
  | }
